godot/src/global.gd

```diff
@@ -91,3 +94,11 @@ func print_node_tree(node: Node, prefix = ""):
 
 func _process(_dt: float):
 	http_requester.poll()
+
+
+func capture_mouse():
```

Comment:
```suggestion
# TODO: check this if there is a better approach for this
# e.g. when the test-runner runs, explorer node doesn't exist (and it shouldn't)
func capture_mouse():
```
add this todo comment or similar, good enough for now

Reply:
now I see the explorer implementation, it could be
```suggestion
func capture_mouse():
  var explorer = get_node_or_null("/root/explorer")
  if is_instance_valid(explorer):
     explorer.capture_mouse()
  else:
	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
```